SQ Plugin: Bump orchestrator to 5.0.0.2065

See #9476, v5 breaks assertions on some ITs, because timestamps are added.

ToDo:
* Check with AE squad if this is expected
* Adjust our assertions if needed

The failing ITs:
```
"07:30:38.596 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]

[ERROR]   ScannerCliTest.givenRazorPagesMainCode_whenScannerForCliIsUsed_logsCSharpWarning:49 
Expecting actual:
  ["07:30:28.121 WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:28.122 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.",
    "07:30:28.123 WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:28.128 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
to contain exactly in any order:
  ["WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.",
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
elements not found:
  ["WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.",
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
and elements not expected:
  ["07:30:28.121 WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:28.122 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.",
    "07:30:28.123 WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:28.128 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]

[ERROR]   ScannerCliTest.givenTestHtmlAndCSharpCode_whenScannerForCliIsUsed_logsCSharpWarning:85 
Expecting actual:
  ["07:30:53.248 WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:53.251 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
to contain exactly in any order:
  ["WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
elements not found:
  ["WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
and elements not expected:
  ["07:30:53.248 WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see [https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",](https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html%22,)
    "07:30:53.251 WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property."]
```

--- a/its/pom.xml
+++ b/its/pom.xml
@@ -27,7 +27,7 @@
     <dependency>
       <groupId>org.sonarsource.orchestrator</groupId>
       <artifactId>sonar-orchestrator-junit5</artifactId>
-      <version>4.9.0.1920</version>
+      <version>5.0.0.2065</version>
     </dependency>
     <dependency>
       <groupId>org.sonarsource.sonarqube</groupId>

--- a/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
+++ b/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
@@ -45,12 +45,14 @@ class ScannerCliTest {
     SonarScanner scanner = getSonarScanner(RAZOR_PAGES_PROJECT, "projects/" + RAZOR_PAGES_PROJECT);
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
-    assertThat(result.getLogsLines(l -> l.contains("WARN")))
+    var logLines = result.getLogsLines(l -> l.contains("WARN"));
+
+    assertThat(logLines)
       .hasSize(4)
       .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
-      .anyMatch(y -> y.endsWith("WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
-      .anyMatch(w -> w.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING))
-      .anyMatch(z -> z.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
+      .anyMatch(y -> y.endsWith("WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"));
+
+    assertThat(logLines.stream().filter(x -> x.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING))).hasSize(2);
 
     // The HTML plugin works
     assertThat(TestUtils.getMeasureAsInt(ORCHESTRATOR, RAZOR_PAGES_PROJECT, "violations")).isEqualTo(2);

--- a/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
+++ b/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
@@ -47,7 +47,6 @@ class ScannerCliTest {
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
       .hasSize(4)
-      .doesNotHaveDuplicates()
       .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
       .anyMatch(y -> y.endsWith("WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
       .anyMatch(w -> w.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING))
@@ -67,7 +66,6 @@ class ScannerCliTest {
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
       .hasSize(2)
-      .doesNotHaveDuplicates()
       .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
       .anyMatch(y -> y.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
 
@@ -85,7 +83,6 @@ class ScannerCliTest {
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
       .hasSize(2)
-      .doesNotHaveDuplicates()
       .anyMatch(y -> y.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
       .anyMatch(z -> z.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
 

--- a/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
+++ b/its/src/test/java/com/sonar/it/shared/ScannerCliTest.java
@@ -37,7 +37,7 @@ class ScannerCliTest {
   private static final String HTML_IN_MAIN_AND_CSHARP_IN_TEST_SUBFOLDERS = "ScannerCli";
 
   // Note: setting the `sonar.projectBaseDir` only enables Incremental PR Analysis when used with the Scanner for .NET.
-  private static final String INCREMENTAL_PR_ANALYSIS_WARNING = "WARN: Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.";
+  private static final String INCREMENTAL_PR_ANALYSIS_WARNING = "WARN  Incremental PR analysis: Could not determine common base path, cache will not be computed. Consider setting 'sonar.projectBaseDir' property.";
 
   @Test
   void givenRazorPagesMainCode_whenScannerForCliIsUsed_logsCSharpWarning() {
@@ -46,12 +46,13 @@ class ScannerCliTest {
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
-      .containsExactlyInAnyOrder(
-        "WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        "WARN: Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        INCREMENTAL_PR_ANALYSIS_WARNING,
-        INCREMENTAL_PR_ANALYSIS_WARNING
-      );
+      .hasSize(4)
+      .doesNotHaveDuplicates()
+      .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(y -> y.endsWith("WARN  Your project contains VB.NET files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(w -> w.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING))
+      .anyMatch(z -> z.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
+
     // The HTML plugin works
     assertThat(TestUtils.getMeasureAsInt(ORCHESTRATOR, RAZOR_PAGES_PROJECT, "violations")).isEqualTo(2);
     TestUtils.verifyNoGuiWarnings(ORCHESTRATOR, result);
@@ -65,10 +66,11 @@ class ScannerCliTest {
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
-      .containsExactlyInAnyOrder(
-        "WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        INCREMENTAL_PR_ANALYSIS_WARNING
-      );
+      .hasSize(2)
+      .doesNotHaveDuplicates()
+      .anyMatch(x -> x.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(y -> y.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
+
     // The HTML plugin works
     assertThat(TestUtils.getMeasureAsInt(ORCHESTRATOR, HTML_IN_MAIN_AND_CSHARP_IN_TEST_SUBFOLDERS, "violations")).isEqualTo(2);
     TestUtils.verifyNoGuiWarnings(ORCHESTRATOR, result);
@@ -82,10 +84,11 @@ class ScannerCliTest {
     BuildResult result = ORCHESTRATOR.executeBuild(scanner);
 
     assertThat(result.getLogsLines(l -> l.contains("WARN")))
-      .containsExactlyInAnyOrder(
-        "WARN: Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html",
-        INCREMENTAL_PR_ANALYSIS_WARNING
-      );
+      .hasSize(2)
+      .doesNotHaveDuplicates()
+      .anyMatch(y -> y.endsWith("WARN  Your project contains C# files which cannot be analyzed with the scanner you are using. To analyze C# or VB.NET, you must use the SonarScanner for .NET 5.x or higher, see https://redirect.sonarsource.com/doc/install-configure-scanner-msbuild.html"))
+      .anyMatch(z -> z.endsWith(INCREMENTAL_PR_ANALYSIS_WARNING));
+
     TestUtils.verifyNoGuiWarnings(ORCHESTRATOR, result);
   }
 

